tests/lean/608.hlean.expected.out

```diff
@@ -1,11 +1,11 @@
 definition id [reducible] : Π {A : Type}, A → A :=
-λ (A : Type) (a : A), a
+λ (A : Type) (H : A), H
```

Comment:
I know what is going on. Floris recently defined `id` at `logic.hlean` like we do in the standard library. The `id` declaration contains the subterm `fun (a : A), a`. Internally, this subterm is represented as `lambda a #0, #0`. The `a` is just a label for pretty printing. This file also contains the term `assume H : a, H` (in the proof of `iff.refl`). Internally, this term is represented as `lambda H #0 #0`. Both terms are identical modulo the label used for pretty printing. When the `.olean` is generated we maximally share the terms. We ignore label information in the process, and both terms are merged. To avoid this kind of issue, we would have to take the names into account when performing the max-sharing step.